src/components/jellyfinActions.ts

```diff
@@ -180,7 +180,7 @@ export function pingTranscoder(
     if (now - lastTranscoderPing < 5000) {
         console.debug('Skipping ping due to recent progress check-in');
         return new Promise(function (resolve) {
-            resolve();
+            resolve(undefined);
         });
```

Comment:
hmm should this be `return Promise.resolve()`?

Reply:
There's a lot of places in this codebase that should probably be doing that... Putting in that undefined cleared out one error and i realized that I probably shouldn't fix the errors in this PR ... honestly didn't think I included this file in the PR but i must of